modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/EditObjectDetails.tsx

```diff
@@ -205,6 +205,13 @@ export default function EditObjectDetails({
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [objectDefinitionId]);
 
+	const onExternalReferenceCodeChange = (externalReferenceCode: string) => {
+		setValues({
+			...values,
```

Comment:
You don´t have to spread values when using the `setValues` returned by `useObjectDetailsForm`. **Please, remove the spread of `values`**. `useObjectDetailsForm` is a custom hook, programmed to easy the state update calls, different from a stock hook like `setState`, where most of the time you will have to use spreads and other magic to ensure a new object has the updated value as well as the old ones.